tests/test_module_caprieval.py

```diff
@@ -225,6 +256,27 @@ def test_protprot_dockq(protprot_caprimodule):
     assert round_two_dec(protprot_caprimodule.dockq) == 0.10
 
 
+def test_protprot_bb_vs_all_atoms(
+    protprot_caprimodule,
+    protprot_allatm_caprimodule,
+        ):
+    """Test difference between all and backbone atoms selection."""
+    assert protprot_caprimodule.allatoms is False
+    assert protprot_allatm_caprimodule.allatoms is True
+    assert protprot_caprimodule.atoms != protprot_allatm_caprimodule.atoms
```

Comment:
I would add something more specific here..maybe on the size of the resulting output